client/network/sync/src/engine.rs

```diff
@@ -848,42 +874,32 @@ where
 			.retain(|stream| stream.unbounded_send(SyncEvent::PeerDisconnected(peer)).is_ok());
 	}
 
+	/// Validate received handshake.
 	fn validate_handshake(
 		&mut self,
-		peer: &PeerId,
+		peer_id: &PeerId,
 		handshake: Vec<u8>,
-	) -> Result<BlockAnnouncesHandshake<B>, ()> {
-		log::trace!(target: LOG_TARGET, "New peer {peer} {handshake:?}");
+	) -> Result<BlockAnnouncesHandshake<B>, bool> {
```

Comment:
I'd expect some documentation about when `Err(true)` and `Err(false)` are returned.

Reply:
I documented it for `NotificationsStreamOpened` but that is wrong place for this documentation, I'll move it to document the function instead.